<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsLayout3DMapWidgetBase</class>
 <widget class="QWidget" name="QgsLayout3DMapWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3D Map</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QLabel" name="mLabel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true">padding: 2px; font-weight: bold; background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="text">
      <string>3D Map</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>510</width>
        <height>635</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="mainLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Scene settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QPushButton" name="mCopySettingsButton">
            <property name="text">
             <string>Copy settings from a 3D view...</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Camera pose</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Center X</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsDoubleSpinBox" name="mCenterXSpinBox">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-9999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Center Y</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QgsDoubleSpinBox" name="mCenterYSpinBox">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-9999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Center Z</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QgsDoubleSpinBox" name="mCenterZSpinBox">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="minimum">
             <double>-9999999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>9999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Distance</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QgsDoubleSpinBox" name="mDistanceToCenterSpinBox">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>9999999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Pitch</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QgsDoubleSpinBox" name="mPitchAngleSpinBox">
            <property name="suffix">
             <string> °</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Heading</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QgsDoubleSpinBox" name="mHeadingAngleSpinBox">
            <property name="suffix">
             <string> °</string>
            </property>
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
